stable/mariadb/values.yaml

```diff
@@ -535,3 +535,11 @@ metrics:
     ## [Kube Prometheus Selector Label](https://github.com/helm/charts/tree/master/stable/prometheus-operator#exporters)
     selector:
       prometheus: kube-prometheus
+
+## Bats Framework (= Bash Automated Testing System) is needed to test if MariaDB is accessible
+## See test-runner.yaml and tests.yaml
+testFramework:
+  image:
+    registry: docker.io
+    repository: dduportal/bats
+    tag: 0.4.0
```

Comment:
There is a _values-production.yaml_ with the same structure than this _values.yaml_ but with some modifications in the default parameters to do it more production-friendly. Can you add those changes at the end of the production file?

Reply:
Ok I will do this.
What would you suggest for this change, should I add the same values on both files?

Reply:
Yes, in this case just copying this code block to other file is fine. Thanks